Docs/NEWS

```diff
@@ -1,114 +1,124 @@
-tvOS 3.0.2:
------------
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## tvOS [Unreleased]
+
+## iOS [Unreleased]
```

Comment:
What do you think about these `unreleased` sections? 

The purpose of them is adding entries to the changelog progressively as soon as feature, fixes, etc... are implemented, getting at the end everything listed at the moment of releasing.

Does this workflow makes sense for vlc-ios?

Reply:
I think they will probably disappear soon since I tend to just copy paste from the line below :D